README.md

```diff
@@ -69,3 +69,5 @@ There are two main dependencies (of course automatically managed by Maven):
 
 The project also has a test scope dependency on [Wiremock](http://wiremock.org), which is really useful in order to avoid hitting the sandbox API all the time.
 
+### Contributors
+- [@erimag](https://github.com/erimag) (Qapital Insight AB)
```

Comment:
Nit: Let's include an empty line after the heading, to respect the document's formatting.